src/main/fc/rc_controls.h

```diff
@@ -101,6 +101,7 @@ typedef struct armingConfig_s {
     uint8_t gyro_cal_on_first_arm;          // allow disarm/arm on throttle down + roll left/right
     uint8_t disarm_kill_switch;             // allow disarm via AUX switch regardless of throttle value
     uint8_t auto_disarm_delay;              // allow automatically disarming multicopters after auto_disarm_delay seconds of zero throttle. Disabled when 0
+    uint8_t small_angle;                    // maximum craft angle from level to allow arming
```

Comment:
`small_angle` is already present in `imuConfig_t` and referenced in:
imu.c:166:
```cpp
smallAngleCosZ = cos_approx(degreesToRadians(imuRuntimeConfig.small_angle));
```
settings.c:523:
```cpp
{ "small_angle",                VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 180 }, PG_IMU_CONFIG, offsetof(imuConfig_t, small_angle) },
```